content/pt-br/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -0,0 +1,266 @@
+---
+reviewers:
+- erictune
+title: Init Containers
+content_type: concept
+weight: 40
```

Comment:
```suggestion
title: Contêineres Init
content_type: concept
weight: 40
```

The Portuguese docs mostly use the term contêineres, not containers.

Reply:
Fixed